apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -66,10 +66,17 @@ function renderSectionProgress(section, validScripts) {
         method: 'GET',
         url: `/dashboardapi/courses?allVersions=1`,
         dataType: 'json'
+      }),
+      // For now, also fetch hidden scripts from the server. This step can be
```

Comment:
Is this tracked somewhere? Would it be helpful to phrase this as a TODO() so we know we have to come back to it.

Reply:
I've reverted this part of the PR, since these scripts were just unhidden in the noon levelbuilder scoop.